solutions/task_6.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python3
+"""
+TASK: Implement at least following functions to execute unittests without errors:
+
+'pair_sum_1':
+    Takes arguments: sequence, expected_sum.
+    Returns all possible pairs (a, b) in the sequence where a + b = expected_sum.
+
+'largest_continuous_sum_1':
+    Takes arguments: sequence.
+    Returns largest sum of continuous sequence among all possible sequences:
+    sum(largest_seq) = max(sum(seq_1), sum(seq_2), sum(seq_3), ..., sum(seq_n))
+
+'convert_sequence_1':
+    Takes arguments: sequence, period.
+    Converts sequence from [a1, a2, a3, b1, b2, b3, c1, c2, c3, ...]
+                       to  [a1, b1, c1, ..., a2, b2, c2, ..., a3, b3, c3, ...]
+    Period has default value = 3.
+
+'get_inf_seq_element_1':
+    Takes arguments: index.
+    Return value with specified index in infinite sequence: [1, 0, 0, 1, 1, 1, 0, 0, 0, 0, 1, 1, 1, 1, 1, ...].
+
+You can implement several solutions for each test and place functions in corresponding lists with functions.
+"""
+import unittest
+
+
+def pair_sum_1(sequence, expected_sum):
+    output = []
+    for first_summand_index in range(len(sequence)):
+        for second_summand_index in range(first_summand_index + 1, len(sequence)):
+            if sequence[first_summand_index] + sequence[second_summand_index] == expected_sum:
+                output.append((sequence[first_summand_index], sequence[second_summand_index]))
+    return output
+
+
+def pair_sum_2(sequence, expected_sum):
+    output = []
+    while len(sequence) > 1:
+        current_value = sequence[0]
+        for index in range(1, len(sequence)):
+            if current_value + sequence[index] == expected_sum:
+                output.append((current_value, sequence[index]))
+                sequence.remove(sequence[index])
+                sequence.remove(current_value)
+                break
+            index += 1
+        if sequence[0] == current_value:
+            sequence.remove(sequence[0])
+    return output
+
+
+def largest_continuous_sum_1(sequence):
+    max_sum = None
+    subsequence_length = len(sequence)
+    while subsequence_length > 0:
+        for subsequence_start_index in range(len(sequence) - subsequence_length + 1):
+            current_sum = sum(sequence[subsequence_start_index:subsequence_start_index + subsequence_length])
+            if max_sum is None or current_sum > max_sum:
+                max_sum = current_sum
+        subsequence_length -= 1
+    return max_sum
+
+
+def convert_sequence_1(sequence, period=3):
+    output = []
+    start_index = 0
+    index_delta = 0
+    while start_index + index_delta < len(sequence):
+        for index_delta in range(0, len(sequence), period):
+            output.append(sequence[start_index + index_delta])
+        start_index += 1
+    return output
+
+
+def convert_sequence_2(sequence, period=3):
```

Comment:
Remove this. The same as above but with argument modifying. 
Do why did you decide that you can modify argument? It's not your's!